### PR TITLE
Revert "delete release notes 8.0"

### PR DESCRIPTION
**Pull request by hfxsd**
Reverts pingcap/docs-cn#16935

### CODE REVIEW

File: releases/release-8.0.0.md [hunk 1]
```diff
@@ -5,9 +5,7 @@ summary: 了解 TiDB 8.0.0 版本的新功能、兼容性变更、改进提升
 
 # TiDB 8.0.0 Release Notes
 
-> **注意：**
->
-> v8.0.0 为 TiDB 尚未发布的版本。以下内容可能在正式发布前发生变化。
+发版日期：2024 年 3 月 28 日
```
Comment (hfxsd) [inline releases/release-8.0.0.md:8]:
```suggestion
发版日期：2024 年 3 月 29 日
```

File: releases/release-timeline.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ summary: 了解 TiDB 的版本发布时间线。
 
 | 版本 | 发布日期 |
 | :--- | :--- |
+| [8.0.0](/releases/release-8.0.0.md) | 2024-03-28 |
```
Comment (hfxsd) [inline releases/release-timeline.md:12]:
```suggestion
| [8.0.0](/releases/release-8.0.0.md) | 2024-03-29 |
```

File: releases/release-notes.md [hunk 3]
```diff
@@ -7,6 +7,10 @@ aliases: ['/docs-cn/dev/releases/release-notes/','/docs-cn/dev/releases/rn/']
 
 TiDB 历史版本发布声明如下：
 
+## 8.0
+
+- [8.0.0-DMR](/releases/release-8.0.0.md): 2024-03-28
```
Comment (hfxsd) [inline releases/release-notes.md:12]:
```suggestion
- [8.0.0-DMR](/releases/release-8.0.0.md): 2024-03-29
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**